lib/text.rb

```diff
@@ -0,0 +1,22 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'dotenv'
+Dotenv.load
+
+class Text
+  
+  def initialize 
+    account_sid = ENV["TWILIO_ACCOUNT_SID"]
+    auth_token = ENV["TWILIO_AUTH_TOKEN"]
+    @client = Twilio::REST::Client.new(account_sid, auth_token)
+  end 
+
+  def send_sms(total, number)
+    @client.messages.create(
+      from: ENV["MY_TWILIO_NUMBER"],
+      to: number,
+      body: "Thank you! Your order was placed and will
+      be delivered #{Time.now + 60 * 60}.")
+  end 
+
+end 
```

Comment:
Twilio code looks good, I like how you have send SMS with arguments which will allow you to input different numbers. 

Does dotenv gem allow you to call on the project specific environment methods with ENV[]?